CMakeLists.txt

```diff
@@ -38,6 +38,13 @@ if (${CMAKE_CXX_COMPILER_ID} STREQUAL MSVC)
     set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} /EHsc")
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /EHsc")
     set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /EHsc")
+else()
+    # Use GNUInstallDirs to set LLVM_LIBDIR_SUFFIX. This should automatically
```

Comment:
besides Linux, this path also handles MacOS. given the variable name is "GNUInstallDirs", does this still works for MacOS?

Reply:
Yep. GNUInstallDirs should always default to "lib" for `CMAKE_INSTALL_LIBDIR` unless the below conditions are met. Just be safe I just tested on a MacOS laptop and it looks like LLVM_LIBDIR_SUFFIX is correctly set to "".

#### GNUInstallDirs.cmake:203
```
  set(_LIBDIR_DEFAULT "lib")
  # Override this default 'lib' with 'lib64' iff:
  #  - we are on Linux system but NOT cross-compiling
  #  - we are NOT on debian
  #  - we are on a 64 bits system
  # reason is: amd64 ABI: https://github.com/hjl-tools/x86-psABI/wiki/X86-psABI
  # For Debian with multiarch, use 'lib/${CMAKE_LIBRARY_ARCHITECTURE}' if
  # CMAKE_LIBRARY_ARCHITECTURE is set (which contains e.g. "i386-linux-gnu"
  # and CMAKE_INSTALL_PREFIX is "/usr"
  # See http://wiki.debian.org/Multiarch
```